digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/RoadAddressService.scala

```diff
@@ -81,13 +82,13 @@ class RoadAddressService(roadLinkService: RoadLinkService, eventbus: DigiroadEve
         roadLinkService.getRoadLinksHistoryFromVVH(floatingAddresses.map(_.linkId).toSet)
       }
     }
-    val historyLinkAddresses = time(logger, "Build history link addresses") {
+    val historyFloatingLinkAddresses = time(logger, "Build history link addresses") {
```

Comment:
I think that we should not change this kind of names from "history link addresses" to "history floating addresses", because we can have history links also without being floating.